library/onotes.inc

```diff
@@ -1,76 +0,0 @@
-<?php
-/**
- *  Office notes library
- *
- * LICENSE: This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 3
- * of the License, or (at your option) any later version.
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
- *
- * @package LibreEHR
- * @author  Brady Miller <brady@sparmy.com>
- * @link    http://librehealth.io
- */
-
-function getOnoteById ($id, $cols = "*")
-{
-	return sqlQuery("select $cols from onotes where id=? order by date DESC limit 0,1", array($id));
-}
-
-function getOnoteByDate ($date, $activity = "1", $cols = "*", $limit="all", $offset="0")
-{
-
-        $sqlBindArray = array();
-	if($limit == "all")
-	{
-		$sql = "select $cols from onotes where date like ? ";
```

Comment:
Remove onotes from database.sql

Reply:
Also add a removal of it from existing databases in the 1_0_1-to-2_0_0_upgrade.sql

Remove it from the menu also